pkgs/python-packages/runes/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage,
+sha256,
+fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "runes";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    repo = "runes";
+    owner = "rustyrussell";
+    rev = "7e3d7648db844ce2c78cc3e9e4f872f827252251";
+    sha256 = "sha256-e0iGLV/57gCpqA7vrW6JMFM0R6iAq5oFwUpZoGySwfs=";
+  };
+
+  propagatedBuildInputs = [ sha256 ];
+
+  meta = with lib; {
+    description = "Runes for authentication (like macaroons only simpler)";
+    homepage = "https://github.com/rustyrussel/runes";
```

Comment:
ACK mod nit

s/rustyrussel/rustyrussell

I would prefer to merge this with a module, but it's ok to merge without for now from my side.